backend/templates/emails/ownership_request.html

```diff
@@ -0,0 +1,17 @@
+<!-- TYPES:
+club_name:
+    type: string
+edit_url:
+    type: string
+full_name:
+    type: string
+-->
+{% extends 'emails/base.html' %}
+
+{% block content %}
+    <h2>Request for ownership for <b>{{ club_name }}</b> from <b>{{ full_name }}</b></h2>
```

Comment:
Nit: might be cleaner to say `"Request for Ownership of {club_name} from {full_name}"`